pages/over-ori-a.md

```diff
@@ -31,9 +31,16 @@ ORI-A deelt het doel van de diverse ORI-initiatieven: het gestandaardiseerd besc
 
 Waar ORI-A verschilt van de andere initiatieven is dat het zich ook richt op het migreren van raadsinformatie. Met ORI-A kan raadsinformatie gestandaardiseerd in XML-formaat worden opgeslagen. Hierin is eerst gekeken naar het ORI-informatiemodel, maar deze kon niet aan alle behoeften van archiefdiensten voldoen. Toen is besloten een archiefvariant van ORI te ontwikkelen. Je kan hier meer over lezen in [Waarom een speciale archiefstandaard?](faq)
 
-# Documentatie
+# Documentatie <!-- nieuwe titel subkop: Legenda website? -->
```

Comment:
Laten we de vraag of een legenda nodig is parkeren tot na de eerste review. 

Mijn mening hierover is dat de zijbalk genoeg als guide moet zijn. Als ie dat niet is dan moet ie worden verbeterd. 

Sowieso lijkt me dit allemaal best wel dubbel op; elke pagina beschrijft nu ook individueel waar ie over gaat.